sparkbox/designsystemssurvey
# Add expires headers

To test:

- Run the site (`npm start`) then, after loading the site, in Dev Tools, click on the Network tab. 
- Check that the `localhost` (or `designsystemssurvey.herokuapp....` if checking on Heroku) file has this under Response Headers: `Cache-Control: public, max-age=0`.
- Check that the `base.css` (or hashed css if on Heroku) file has this under Response Headers: `Cache-Control: public, max-age=86400`
- Check that the `main.js` (or hashed js if on Heroku) file has this under Response Headers: `Cache-Control: public, max-age=86400`
